package.json

```diff
@@ -103,6 +103,11 @@
                     "type": "boolean",
                     "default": true,
                     "description": "Use GitHub for creating new repository"
+                },
+                "deployToAzure.UseRemoteConfigurer": {
```

Comment:
I don't think we should have this as a configuration setting? It is confusing for user to understand this variable.
We should just use a global variable for this. If customers report problem, we will set this variable to false and release a new version of vs code.

Reply:
Okay, I will remove it from here and keep in extension variables and set to value to true

Reply:
done